### DISCUSSION TITLE
Quick Start Guide

### DISCUSSION THREAD

**Discussion by CaMer0n** [e107inc/e107#4752] [Q&A Installation]
I would like to get everyone's feedback on the first steps they typically take once the e107 Installation process has completed. 
Eventually like to have it included in a Administrator Quick Start guide. Thanks. 

1. Review all areas and fields in `Settings > Preferences` and modify as needed. 
2. Send a test email `Settings > Preferences Email & Contact Info > Send Test Email to: ____ `
3. Enable Search Engine Friendly (SEF) URLs on News, Pages, etc. ` Settings > URL Configuration > Profiles`
4. Change Site Theme (and Skin if needed) `Manage > Theme Manager > Change Theme`
5. Install Google Sitemap `Manage > Plugin Manager > Not Installed`
6. Install Linkwords `Manage > Plugin Manager  > Not Installed`

....

**Comment by Jimmi08** [comment]
1 - 4 
checking if Tinymce is installed

**Comment by chiabgigi** [comment]
I would like to share the folder: e107_languages ​​/ Italian what can I do?

**Reply by kreossino** [reply]
clone the repository git in your works area and commit and push all files modified with git push command
this is a repo
git@github.com:e107translations/Italian.git

.....
https://linuxhub.it/articles/howto-utilizzo-di-ssh-per-connettersi-a-github/

**Reply by chiabgigi** [reply]
ciao creossino
grazie per la risposta, non sono molto erudito con git quindi ti rimando due link sperando che tu faccia meglio di me:

https://github.com/chiabgigi/e107_languages-Italian

https://drive.google.com/drive/folders/1MeHXoAF-xjf6e0YZiO6qeP5Z2kKEX1e-?usp=sharing

scusa ancora per la mia ignoranza
ciao
